src/styles/variables.json

```diff
@@ -1206,6 +1206,9 @@
             "gap": "0.75rem"
           }
         },
+        "size": {
+          "minWidth": "180px"
```

Comment:
is this an acceptable value?

Reply:
@gjones Should we reduce it?

Reply:
Yeah @fneves, this seems like a solid enough place to start regarding min-width, I tested out on a few menus and seemed decent, we'll probably want something more dynamic in the future though, sm, md, lg, or something.